src/pharmpy/tools/structsearch/drugmetabolite.py

```diff
@@ -4,6 +4,28 @@
 from pharmpy.modeling import add_metabolite, add_peripheral_compartment, set_name
 
 
+def create_base_drug_metaboltie(model: Model) -> Model:
```

Comment:
misspelled function name

Reply:
Oh how embarrassing! I corrected the error.